li-utils/src/main/pegasus/com/linkedin/common/MLFeatureDataType.pdl

```diff
@@ -80,4 +80,19 @@ enum MLFeatureDataType {
   * Set Data Type ex: set, frozenset
   */
   SET
+
+  /**
+   * Continuous data are made of uncountable values, often the result of a measurement such as height, weight, age etc.
```

Comment:
Are these always numeric? Are they always decimal numbers or something whole numbers, both? I'm guessing either or

Reply:
Yep. Technically these would be rational ones I guess